Fix the capture of QSVT (#7360)
PennyLaneAI/pennylane: doc/releases/changelog-dev.md

<h3>Bug fixes 🐛</h3>

* With program capture enabled (`qml.capture.enable()`), `QSVT` no treats abstract values as metadata.

>   [(#7360)](https://github.com/PennyLaneAI/pennylane/pull/7360)

Author: ```suggestion
```

<h3>Bug fixes 🐛</h3>

> * Fixes the capture of `QSVT` so that it no longer treats abstract values as metadata.

Author: ```suggestion
* Fixes the capture of `QSVT` so that it no longer treats abstract values as metadata.
  [(#7360)](https://github.com/PennyLaneAI/pennylane/pull/7360)
```

Reviewer: @albi3ro was this fix made purely for compatibility with JAX>0.4.28?

Reviewer: ```suggestion
* Fixed a bug that occurred with `qml.QSVT` and program capture enabled (`qml.capture.enable`), where abstract values were being treated as metadata.
```

Author: ```suggestion
* With program capture enabled (`qml.capture.enable()`), `QSVT` no treats abstract values as metadata.
  [(#7360)](https://github.com/PennyLaneAI/pennylane/pull/7360)
```
